lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/core/impl/qm/XwikiBridgeInterfaceRestResource.java

```diff
@@ -22,51 +22,61 @@
 import org.xwiki.component.annotation.Component;
 import org.xwiki.rest.XWikiRestComponent;
 
-import eu.learnpad.exception.LpRestException;
-import eu.learnpad.exception.impl.LpRestExceptionImpl;
+import eu.learnpad.exception.impl.LpRestExceptionXWikiImpl;
 import eu.learnpad.qm.BridgeInterface;
 import eu.learnpad.core.rest.RestResource;
 
+/*
+ * The methods inherited form the BridgeInterface in this
+ * class should be implemented as a REST invocation
+ * toward the BridgeInterface binded at the provided URL
+ */
 @Component
```

Comment:
Btw I am reconsidering the class "eu.learnpad.core.impl.qm.XwikiBridgeInterfaceRestResource.java". The responsability of this class is to hide a Controller the fact that BridgeInterface is actually exposed as REST somewhere else (i.e. another host). Thus I am not sure anymore the annotation if @Component make it sense there. In other words, that class should only extend RestResources and implements BridgeInterface, but it has not implement org.xwiki.rest.XWikiRestComponent. What do you think?